### ISSUE TITLE
https://captum.ai/tutorials/CIFAR_TorchVision_Interpret error int line 205

### ISSUE THREAD

**Issue by nandanii** [pytorch/captum#1153]
## 🐛 Bug

_ = viz.visualize_image_attr(None, original_image, method="original_image", title="Original Image")
## To Reproduce

ValueError                                Traceback (most recent call last)
[<ipython-input-46-120daaf77f04>](https://localhost:8080/#) in <cell line: 1>()
----> 1 _ = viz.visualize_image_attr(None, original_image, method="original_image", title="Original Image")

4 frames
[/usr/local/lib/python3.10/dist-packages/captum/attr/_utils/visualization.py](https://localhost:8080/#) in visualize_image_attr(attr, original_image, method, sign, plt_fig_axis, outlier_perc, cmap, alpha_overlay, show_colorbar, title, fig_size, use_pyplot)
    248     plt_axis.set_yticklabels([])
    249     plt_axis.set_xticklabels([])
--> 250     plt_axis.grid(b=False)
    251 
    252     heat_map = None

[/usr/local/lib/python3.10/dist-packages/matplotlib/axes/_base.py](https://localhost:8080/#) in grid(self, visible, which, axis, **kwargs)
   3192         _api.check_in_list(['x', 'y', 'both'], axis=axis)
   3193         if axis in ['x', 'both']:
-> 3194             self.xaxis.grid(visible, which=which, **kwargs)
   3195         if axis in ['y', 'both']:
   3196             self.yaxis.grid(visible, which=which, **kwargs)

[/usr/local/lib/python3.10/dist-packages/matplotlib/axis.py](https://localhost:8080/#) in grid(self, visible, which, **kwargs)
   1658             gridkw['gridOn'] = (not self._major_tick_kw['gridOn']
   1659                                 if visible is None else visible)
-> 1660             self.set_tick_params(which='major', **gridkw)
   1661         self.stale = True
   1662 

[/usr/local/lib/python3.10/dist-packages/matplotlib/axis.py](https://localhost:8080/#) in set_tick_params(self, which, reset, **kwargs)
    930         """
    931         _api.check_in_list(['major', 'minor', 'both'], which=which)
--> 932         kwtrans = self._translate_tick_params(kwargs)
    933 
    934         # the kwargs are stored in self._major/minor_tick_kw so that any

[/usr/local/lib/python3.10/dist-packages/matplotlib/axis.py](https://localhost:8080/#) in _translate_tick_params(kw, reverse)
   1074         for key in kw_:
   1075             if key not in allowed_keys:
-> 1076                 raise ValueError(
   1077                     "keyword %s is not recognized; valid keywords are %s"
   1078                     % (key, allowed_keys))

ValueError: keyword grid_b is not recognized; valid keywords are ['size', 'width', 'color', 'tickdir', 'pad', 'labelsize', 'labelcolor', 'zorder', 'gridOn', 'tick1On', 'tick2On', 'label1On', 'label2On', 'length', 'direction', 'left', 'bottom', 'right', 'top', 'labelleft', 'labelbottom', 'labelright', 'labeltop', 'labelrotation', 'grid_agg_filter', 'grid_alpha', 'grid_animated', 'grid_antialiased', 'grid_clip_box', 'grid_clip_on', 'grid_clip_path', 'grid_color', 'grid_dash_capstyle', 'grid_dash_joinstyle', 'grid_dashes', 'grid_data', 'grid_drawstyle', 'grid_figure', 'grid_fillstyle', 'grid_gapcolor', 'grid_gid', 'grid_in_layout', 'grid_label', 'grid_linestyle', 'grid_linewidth', 'grid_marker', 'grid_markeredgecolor', 'grid_markeredgewidth', 'grid_markerfacecolor', 'grid_markerfacecoloralt', 'grid_markersize', 'grid_markevery', 'grid_mouseover', 'grid_path_effects', 'grid_picker', 'grid_pickradius', 'grid_rasterized', 'grid_sketch_params', 'grid_snap', 'grid_solid_capstyle', 'grid_solid_joinstyle', 'grid_transform', 'grid_url', 'grid_visible', 'grid_xdata', 'grid_ydata', 'grid_zorder', 'grid_aa', 'grid_c', 'grid_ds', 'grid_ls', 'grid_lw', 'grid_mec', 'grid_mew', 'grid_mfc', 'grid_mfcalt', 'grid_ms']

## Environment
Describe the environment used for Captum

```

 - Captum / PyTorch Version (e.g., 1.0 / 0.4.0):
 - OS (e.g., Linux):
 - How you installed Captum / PyTorch (`conda`, `pip`, source):
 - Build command you used (if compiling from source):
 - Python version:
 - CUDA/cuDNN version:
 - GPU models and configuration:
 - Any other relevant information:

## Additional context

<!-- Add any other context about the problem here. -->

**Comment by LostOxygen** [comment]
Got already fixed in PR #1118 on the main branch.

**Comment by vivekmig** [comment]
Hi @nandanii , as @LostOxygen mentioned, we have fixed this issue in https://github.com/pytorch/captum/pull/1118 , but this fix is not yet in a release package. In the meantime, you can install Captum from source (master) as follows to avoid this issue:
```
pip install git+https://github.com/pytorch/captum.git
```
